pkg/operator/ceph/csi/spec.go

```diff
@@ -96,7 +96,7 @@ var (
 // manually challenging.
 var (
 	// image names
-	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.0.0"
+	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.1.0"
```

Comment:
same change need to be done for helm charts, operator.yaml openshift-operator.yaml and also need to updated the upgrade-documentation also